app/scripts/release.js

```diff
@@ -19,8 +19,6 @@ async function main() {
     if (error.stderr.match("command not found")) {
       console.log("Please install missing dependencies:");
       console.log("  - https://github.com/github/hub");
-    } else {
-      raise;
```

Comment:
```suggestion
throw(error);
```
Hehe, looks like I'm still writing Ruby in my mind 🤦‍♂

Reply:
`throw error` actually ;-)